### PR TITLE
Use metadata params for submission

### PR DESCRIPTION
**Pull request by RobbinBouwmeester**

### CODE REVIEW

File: proteobench/modules/dda_quant/module.py [hunk 1]
```diff
@@ -370,9 +384,33 @@ def write_intermediate_raw(
...
+
+    def load_params_file(
+        self, input_file: str, input_format: str
+    ) -> ProteoBenchParameters:
+        """Method loads parameters from a metadata file depending on its format. TODO: Currently only supports MaxQuant, MSFragger, and Proline"""
+        params = ProteoBenchParameters()
+
+        if input_format == "MaxQuant":
+            params = maxquant.extract_params(input_file)
+        # elif input_format == "AlphaPept":
```
Comment (enryH) [inline proteobench/modules/dda_quant/module.py:403]: also availabe. Would you like to have a dictionary of `extract_params` functions in the module `preteobench.io.params`? Then you could avoid the long block of if/else statements and handle potential key-errros.

**Comment by scaramonche** [comment]
@RobbinBouwmeester I think it is pretty much done. Tests should work once the results.json in the repo is replaced. Note that the biggest change comes from using `software_name` as the variable that contains the `input_format` (MaxQuant, etc.) instead of `search_engine` as used until now. 
I did not change the template yet, but I can also do that later.

File: proteobench/modules/dda_quant/module.py [hunk 2]
```diff
@@ -343,6 +359,8 @@ def clone_pr(
         current_datapoint = temporary_datapoints.iloc[-1]
         current_datapoint["is_temporary"] = False
         for k, v in datapoint_params.__dict__.items():
+            if k == "software_name":
```
Comment (RobbinBouwmeester) [inline proteobench/modules/dda_quant/module.py:362]: This needs to be removed?

File: proteobench/modules/dda_quant/module.py [hunk 3]
```diff
@@ -380,6 +398,8 @@ def write_json_local_development(self, temporary_datapoints, datapoint_params):
 
         # Update parameters based on parsed params
         for k, v in datapoint_params.__dict__.items():
+            if k == "software_name":
```
Comment (RobbinBouwmeester) [inline proteobench/modules/dda_quant/module.py:401]: This needs to be removed?

File: proteobench/modules/dda_quant/module.py (hunk 2 above)
Comment (scaramonche) [reply proteobench/modules/dda_quant/module.py]: Indeed, fixed in a (hopefully) more elegant way.

**Review by scaramonche — APPROVED** [review]
I think we are good to go. Maybe @enryH can have a final look as an independent observer?

**Pull request merged**